packages/service/src/event-stream/model/events.ts

```diff
@@ -51,6 +56,7 @@ type EventData =
   | PauseEvent
   | ResumeEvent
   | SkipEvent
+  | SpeedEvent
```

Comment:
żeby uspójnić to 
  SpeedEvent -> ChangeSpeedEvent 
albo 
  ChangeVolumeEvent -> VolumeEvent. 
WDYT?